cfpb/cfgov-refresh
# Housing counselor: Add responsive styling for search results header

Fixes [GHE]/CFGOV/platform/issues/3190

## Changes

- Remove `float:right` at mobile screen sizes for search result header and add left margin to links.

## Testing

1. `./frontend.sh`
2. Load find-a-housing counselor page and search for a ZIP code.
3. Scroll down to results header and resize down to mobile size.
4. See that results header text and links stack at small sizes.

## Screenshots

### Before:

<img width="325" alt="screen shot 2018-12-11 at 9 16 23 am" src="https://user-images.githubusercontent.com/704760/49806482-ddb45380-fd25-11e8-92a5-600e29a4c253.png">

---

### After:

<img width="880" alt="screen shot 2018-12-11 at 9 13 20 am" src="https://user-images.githubusercontent.com/704760/49806510-f15fba00-fd25-11e8-8866-dab4ca44c5e7.png">

---

<img width="714" alt="screen shot 2018-12-11 at 9 13 35 am" src="https://user-images.githubusercontent.com/704760/49806511-f15fba00-fd25-11e8-9b54-49f9483ed5b9.png">

---

<img width="714" alt="screen shot 2018-12-11 at 9 13 45 am" src="https://user-images.githubusercontent.com/704760/49806512-f1f85080-fd25-11e8-95b9-8ee3ebf5fccd.png">

---

<img width="570" alt="screen shot 2018-12-11 at 9 14 01 am" src="https://user-images.githubusercontent.com/704760/49806514-f1f85080-fd25-11e8-972e-961621e54d5d.png">

---

<img width="573" alt="screen shot 2018-12-11 at 9 14 08 am" src="https://user-images.githubusercontent.com/704760/49806515-f1f85080-fd25-11e8-8ad9-84db2b7d8de6.png">

---

<img width="306" alt="screen shot 2018-12-11 at 9 14 26 am" src="https://user-images.githubusercontent.com/704760/49806516-f290e700-fd25-11e8-93b4-987ec40b6632.png">
